engine/src/main/java/com/redhat/cloud/notifications/processors/email/EmailSubscriptionTypeProcessor.java

```diff
@@ -296,10 +280,10 @@ private void processAggregateEmailsByAggregationKey(EmailAggregationKey aggregat
 
                 emailSender.sendEmail(aggregation.getKey(), event, subject, body);
```

Comment:
This will never throw any exception raised during the email sending. They are catched in `emailSender`.